sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -245,11 +245,13 @@ def delete_experiment(self, name, namespace=None):
                 "Exception when calling CustomObjectsApi->delete_namespaced_custom_object:\
          %s\n" % e)
 
-    def list_experiments(self, namespace=None):
+    def list_experiments(self, namespace=None, in_short=False):
         """List all Katib Experiments.
 
         :param namespace: Experiments namespace.
         If the namespace is None, it takes "default" namespace.
+        :param in_short: List Experiments in summary or not
+        If the in_short is True, it will shows only name and status of each Experiments.
```

Comment:
Do we really need `in_short` parameter ? Is it common practice to have this functionality with Kubernetes API ?
What do you think @johnugeorge @gaocegege @anencore94 ?

Reply:
In fact, I was wondering whether this feature is necessary or not too.
But since I'm not sure why the original api returns only name and status, I just thought there must be some reason before and some people still wants to use api with such responses. (to match backward compatibility)
However, I'm afraid it would be not that common usage

Reply:
For now, I removed `in_short` parameter since I couldn't find any practice to use such parameter within k8s modules

Reply:
Nice suggestion about compatibility.
Maybe we can log that `in_short` parameter will be deprecated in the next SDK releases and just remove it after 1-2 SDK release?
What do you think ?

Reply:
Yeap, that would be better to care someone who wants to list experiments only with name and status as before.

By the way, 
>Maybe we can log that in_short parameter will be deprecated in the next SDK releases and just remove it after 1-2 SDK release?

Did you mean `log those` on docstring? or real log ? @andreyvelich

Reply:
I was thinking about including the log message somewhere here: https://github.com/anencore94/katib/blob/778e4fdf8d995a00f37ba06e4e5c3b768152f1ed/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L274-L279.

Maybe it's redundant, what do others think @gaocegege @johnugeorge ?

Reply:
Maybe just deprecate old API style with sufficient explanation in release note would be fine.